cmd/goQuery/commands/root.go

```diff
@@ -36,11 +37,9 @@ var subRootCmd = &cobra.Command{}
 
 // Execute is the main entrypoint and runs the CLI tool
 func Execute() {
-	err := rootCmd.Execute()
-	if err != nil {
-		os.Exit(1)
+	if err := rootCmd.Execute(); err != nil {
+		logging.Logger().Fatalf("Error running query: %s", err)
```

Comment:
With the recent changes in https://github.com/els0r/goProbe/pull/108, we'd just instantiate a "plain" CLI logger_ before the if and then call that one.